tests/test_functions.py

```diff
@@ -434,8 +434,25 @@ def test_sim2sumo_with_ert(scratch_files, case_uuid, sumo, monkeypatch):
     real0 = scratch_files[0]
     write_ert_config_and_run(real0)
     expected_exports = 88
-    path = f"/objects('{case_uuid}')/children"
-    results = sumo.get(path).json()
+    path = f"/objects('{case_uuid}')/search"
+    results = sumo.post(
+        path,
+        json={
+            "query": {
+                "bool": {
+                    "must": [{"term": {"fmu.case.uuid.keyword": case_uuid}}],
```

Comment:
Don't actually need this, as the this particular endpoint adds a restriction on `case_uuid`.

Reply:
right.. 🤦